docs/langref.md

```diff
@@ -262,6 +262,7 @@ Example := Window {
 ```
 
 * **`length`**: One can query the length of an array and model using the builtin `.length` property.
+* **`array[index]`**: Individual elements of an array can be retrieved using the `array[index]` syntax.
```

Comment:
should we specify what happens when we go out of bounds?  should we have examples in the documentation?